src/index.ts

```diff
@@ -153,6 +153,20 @@ export const combineValidators =
     }
   };
 
+export const toOptionalValidator =
+  <Value, ErrorMessage = string, EmptyValue extends Value = Value>(
```

Comment:
can we rather get an `isEmptyValue: (x: Value) => boolean`? that'd help for values without referential equality.

Reply:
Done https://github.com/swan-io/react-ux-form/pull/25/commits/ab1c5dc5e0d4874f9d6f3a2f9c8a396a9fa002d6